src/sysupdate/org.freedesktop.sysupdate1.policy

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?> <!--*-nxml-*-->
+<!DOCTYPE policyconfig PUBLIC "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+        "https://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
+
+<!--
+  SPDX-License-Identifier: LGPL-2.1-or-later
+
+  This file is part of systemd.
+
+  systemd is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+-->
+
+<policyconfig>
+
+        <vendor>The systemd Project</vendor>
+        <vendor_url>https://systemd.io</vendor_url>
+
+        <!--
+            SECURITY: the default policy allows any user with an active session on the local console to check
+            for updates and update the system to the latest version without extra authentication.
+            Depending on the use case it might make sense to request authentication here, or add a polkit
+            rule to only allow access to these actions for members of a given group.
+        -->
+
+        <action id="org.freedesktop.sysupdate1.check">
+                <description gettext-domain="systemd">Check for system updates</description>
+                <message gettext-domain="systemd">Authentication is required to check for system updates</message>
+                <defaults>
+                        <allow_any>auth_admin</allow_any>
+                        <allow_inactive>auth_admin</allow_inactive>
+                        <allow_active>yes</allow_active>
+                </defaults>
+        </action>
+
+        <action id="org.freedesktop.sysupdate1.update">
```

Comment:
I can't see the conversation you're linking to. Do you want to move the comment https://github.com/CodethinkLabs/systemd/blob/7d95d71d53f2be5245f88de66c3602b793bebd57/src/sysupdate/org.freedesktop.sysupdate1.policy#L21-L26 here, or do you want an additional comment?

Reply:
Somehow missed that comment... But yeah, I would add examples of prior art to explain why it's done like this, since the question was brought up during review

Reply:
I've expanded that comment to include notes on prior art.